src/mlpack/methods/ann/gan/gan_impl.hpp

```diff
@@ -42,7 +42,6 @@ GAN<Model, InitializationRuleType, Noise, PolicyType>::GAN(
     const double multiplier,
     const double clippingParameter,
     const double lambda):
-    predictors(predictors),
```

Comment:
Please revert this change. We wish to alias `GAN` predictors as `Discriminator` predictors.